### PR TITLE
bpo-45855: Replaced deprecated `PyImport_ImportModuleNoBlock` with PyIm…

### PR DESCRIPTION
**Pull request by kumaraditya303**
…port_ImportModule

https://bugs.python.org/issue45855

### CODE REVIEW

**Comment by kumaraditya303** [comment]
cc @asvetlov

File: Misc/NEWS.d/next/C API/2021-12-11-08-41-36.bpo-45855.Lq2_gR.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Replaced deprecated :c:func:`PyImport_ImportModuleNoBlock` with :c:func:`PyImport_ImportModule`. Patch by Kumar Aditya.
```
Comment (arhadthedev) [inline Misc/NEWS.d/next/C API/2021-12-11-08-41-36.bpo-45855.Lq2_gR.rst]: Such a wording may create an impression that the whole Python (C API included) makes a replacement.

```suggestion
Built-in Python modules stopped using deprecated :c:func:`PyImport_ImportModuleNoBlock` in favor of :c:func:`PyImport_ImportModule`. Patch by Kumar Aditya.
```

**Review by arhadthedev — CHANGES_REQUESTED** [review]

**Review by tiran — APPROVED** [review]
Thanks, LGTM! 

PyImport_ImportModuleNoBlock is basically an alias for PyImport_ImportModule nowadays.

```
PyObject *
PyImport_ImportModuleNoBlock(const char *name)
{
    return PyImport_ImportModule(name);
}
```

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**